Apps/phjira/jira_connector.py

```diff
@@ -794,6 +795,7 @@ def _add_comment(self, param):
         issue_id = param[JIRA_JSON_ID]
 
         comment_body['body'] = param[JIRA_JSON_COMMENT]
+        # comment_body['properties'] = [{'key': 'sd.public.comment', 'value': {'internal': "True"}}]
```

Comment:
Is this supposed to be commented out or removed?

Reply:
Removed